src/10_functions.py

```diff
@@ -2,11 +2,22 @@
 
 # YOUR CODE HERE
 
+
+def is_even(num):
+    if num % 2 == 0:  # if the number divided by two and has no remainder... return true
+        return True
+    else:  # else return false
+        return False
+
+
 # Read a number from the keyboard
 num = input("Enter a number: ")
 num = int(num)
 
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
 # YOUR CODE HERE
-
+if (num % 2 == 0):
```

Comment:
more DRY to use your "is_even" function to do this, maybe even some ternary syntax